Application port Tutorial 6 merge fix

byebrid opened the pull request:
- Fix the merge resulting from #8 .
- Problem was home page being out of date in resultant merge, fixed to add tutorial buttons back in.

byebrid committed:
Merge remote-tracking branch 'origin/ApplicationPortTutorial6' into ApplicationPort

MartyCarroll commented:
Works fine, just need to change title (shows tutorial 2 for me)
The instructions don't work properly on iOS but merge it anyway and we can do a system wide refactor next semester

byebrid committed:
Correctly display tutorial 6 and keyboard modules

- Suey had already fixed these in c21a218 and 191d4db, I had just
  stuffed up the merge fix so bringing them back.
